icons/picoctf.svg

```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" role="img" viewBox="0 0 24 24">

```

Comment:
You will need to remove this extraneous attribute `xmlns:xlink="http://www.w3.org/1999/xlink"`